cli/scorecard/inventory.go

```diff
@@ -61,11 +62,12 @@ func TargetOrg(organizationID string) Option {
 }
 
 // NewInventory creates a new CAI inventory manager
-func NewInventory(bucketName, dirPath string, readFromStdin bool, refresh bool, options ...Option) (*InventoryConfig, error) {
+func NewInventory(bucketName, dirPath string, readFromStdin bool, refresh bool, concurrency bool, options ...Option) (*InventoryConfig, error) {
```

Comment:
`concurrency` should be added as an Option (using the option pattern above) instead of an parameter.